Tests.Integration.Presentation.Web/ItSystem/SystemRelationsTest.cs

```diff
@@ -366,6 +367,40 @@ public async Task Can_Get_SystemRelations_AssociatedWithContract()
             }
         }
 
+        [Fact]
+        public async Task Can_Get_SystemRelations_Defined_In_Organization()
+        {
+            //Arrange
+            var input1 = await PrepareFullRelationAsync(false, false, false);
+            var input2 = await PrepareFullRelationAsync(false, false, false);
+            var input3 = await PrepareFullRelationAsync(false, false, false);
+
+            //Make sure there are not relations before performing the paging test
+            DatabaseAccess.MutateEntitySet<SystemRelation>(repository =>
+            {
+                var systemRelations = repository.AsQueryable().ToList();
+                repository.RemoveRange(systemRelations);
+                repository.Save();
+            });
+
+            var relation1 = await SystemRelationHelper.PostRelationAsync(input1);
+            var relation2 = await SystemRelationHelper.PostRelationAsync(input2);
+            var relation3 = await SystemRelationHelper.PostRelationAsync(input3);
+
+            var expectedSequence = new[] { relation1.Id, relation2.Id, relation3.Id }.OrderBy(id => id).ToList();
+
+            //Act
+            const int pageSize = 2;
+            var firstPage = await SystemRelationHelper.GetRelationsDefinedInOrganization(TestEnvironment.DefaultOrganizationId, 0, pageSize);
+            var secondPage = await SystemRelationHelper.GetRelationsDefinedInOrganization(TestEnvironment.DefaultOrganizationId, 1, pageSize);
+
+            //Assert - first page with 2 items
+            Assert.Equal(expectedSequence.Take(2), firstPage.Select(x => x.Id));
```

Comment:
Ville det ikke være mere korrekt at bruge "pageSize" frem for "2"?

Reply:
enig